src/app/patient-info/page.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import Image from 'next/image';
+import dynamic from 'next/dynamic';
 import PatientViewNavBar from '../navbar/navbar-patient'
 import InfoPanel from '../info-panel/info-panel'
 import './patient-info.css'
 
-var img = require('../../assets/tumor.jpg');
+const VisualizationComponent = dynamic(
```

Comment:
my personal preference is that we put functions inside of the main function on a tsx page

Reply:
otherwise i would move it to a different file

Reply:
i think this with the dynamic component is standard for next tho so ignore this but just fyi general thoughts